content/reference/endpoints/inquiries.md

```diff
@@ -46,6 +45,7 @@ lastname         | String  | Read/Write | **Required**. Inquiry's client lastnam
 message          | Text    | Read/Write | Inquiry's message. (500 characters max)
 phone            | String  | Read/Write | Inquiry's client phone number.
 source_id        | Integer | Write      | Source id related to the Inquiry.
+rental_id        | Integer | Write      | **Only update** Id of rental related to the Inquiry
```

Comment:
I think those ids should be moved to `id` section, thats how we started to do it for `bookings_write` http://developers.bookingsync.com/reference/endpoints/bookings/